data/stoa0045a/__cts__.xml

```diff
@@ -1,3 +1,3 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:stoa0045a">
-  <ti:groupname xml:lang="eng">Catholic Church Pope</ti:groupname>
+  <ti:groupname xml:lang="eng">Catholic Church. Pope</ti:groupname>
```

Comment:
This is `Catholic Church. Pope` here and `Catholic Church Pope` below.

Reply:
I will fix that.